Gemfile.lock

```diff
@@ -345,39 +314,10 @@ GEM
       wmi-lite (~> 1.0)
     parser (2.4.0.0)
       ast (~> 2.2)
-    plist (3.2.0)
+    plist (3.3.0)
     poise (2.7.2)
       halite (~> 1.0)
-    poise-boiler (1.13.2)
-      bundler
-      chefspec (~> 5.0)
-      codeclimate-test-reporter (~> 0.4)
-      codecov (~> 0.0, >= 0.0.2)
-      foodcritic (>= 7, < 9)
-      fuubar (~> 2.0)
-      git (~> 1.2)
-      halite (~> 1.2)
-      kitchen-docker (>= 2.6.0.rc.0)
-      kitchen-ec2 (~> 1.0)
-      kitchen-sync (~> 2.1)
-      kitchen-vagrant
-      mixlib-shellout (>= 1.4, < 3.0)
-      poise-profiler (~> 1.0)
-      pry
-      pry-byebug
-      rake (>= 10.4, < 12.0)
-      rspec (~> 3.2)
-      rspec-its (~> 1.2)
-      simplecov (~> 0.9)
-      test-kitchen (~> 1.7, >= 1.7.1)
-      travis (~> 1.8, >= 1.8.1)
-      vagrant-wrapper
-      winrm (>= 1.6, < 3)
-      winrm-fs (>= 0.4, < 2)
-      yard (~> 0.8)
-      yard-classmethods (~> 1.0)
-    poise-profiler (1.0.1)
-      halite (~> 1.0)
+    poise-boiler (0)
```

Comment:
I should go yank that version, thanks for the reminder :) I use a `0` version as a placeholder when reserving names but it can allow bad version solutions.